src/Tests/Microsoft.NET.Build.Tests/GlobalPropertyFlowTests.cs

```diff
@@ -248,11 +235,13 @@ public void TestGlobalPropertyFlowInSolution(bool passSelfContained, bool passRu
             }
         }
 
-        private static void ValidateProperties(TestAsset testAsset, TestProject testProject, bool expectSelfContained, bool expectRuntimeIdentifier, string targetFramework = null, string expectedRuntimeIdentifier = "")
+        private static void ValidateProperties(TestAsset testAsset, TestProject testProject, bool expectSelfContained, bool expectRuntimeIdentifier, string thisTargetFramework = null, string expectedRuntimeIdentifier = "")
```

Comment:
The parameter rename here is because the IDE was getting confused when doing type checking on which targetFramework property was which.